lib/ProductOpener/Users.pm

```diff
@@ -680,7 +775,7 @@ sub init_user() {
 		# If the user exists
 		if (defined $user_id) {
 
-           my  $user_file = "$data_root/users/" . get_string_id_for_lang("no_language", $user_id) . ".sto";
+           my $user_file = "$data_root/users/" . get_string_id_for_lang("no_language", $user_id) . ".sto";
```

Comment:
We use it at several places, I would find it better to have it in a function (get_user_file).

But that's for another refactor ;-)

Reply:
> But that's for another refactor ;-)

Yes :) This refactor broke a lots of things, I need to update all cgi scripts to use my $request_ref = init_request() instead of init(). I'll first repair everything I broke.